README.adoc

```diff
@@ -15,8 +15,10 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* Ark is a desktop delivery management application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java application intended to help delivery manager manage the parcels they need to deliver.
+* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
+* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
+* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
+* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
```

Comment:
I think the changes to the documentation, xml and .gitignore files should be removed from the PR, i think they were unintended